packages/shared/src/components/fields/MarkdownInput/index.tsx

```diff
@@ -150,6 +152,16 @@ function MarkdownInput(
     }
   };
 
+  useEffect(() => {
+    const content = textareaRef?.current?.value;
+
+    if (!content) {
+      return;
+    }
+
+    focusInput(textareaRef.current, [content.length, content.length]);
+  }, []);
```

Comment:
I see more and more hacks with inputs like this due to not having reactive state instance in recent times. I get it we don't have other option now but it looks like a lot of tech debt.

Reply:
Very valid point, for now lets proceed, ill add it as discussion topic for web. Around moving to form lib. (See ido also mentioned one)

Is that ok?

Reply:
Yes of course, don't think of it as blocking.

Reply:
Most likely a form library.
There are some suggestions floating for instance https://tanstack.com/form/latest

Reply:
I see. Yep. I would be glad if we could use any available packages like that, or `react-form-hooks` can be a good option as well. We can totally refactor our input components by then.